### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by username-forGithub**
Updated index.html, style.css, main.js
Added popup window

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Project name
-> Portfolio Mobile Menu
+> Portfolio: popup window
```
Comment (Bondok6) [inline README.md:2]: - [ ] I can see that you're using the Microverse Template Great Job 👌💯 but I suggest adding a description to your project and the rest of the sections such as the `Get Started`, and `License` sections. Please check [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project 👌

File: css/style.css [hunk 2]
```diff
@@ -882,3 +905,141 @@ footer .line {
...
+  border: none;
+  font-weight: 600;
+  font-family: "Inter", sans-serif;
+  margin-left: 12px;
+  margin-top: 0;
+  white-space: nowrap;
+  letter-spacing: initial;
+  padding-left: 20px;
+  padding-right: 20px;
+}
```
Comment (Bondok6) [inline css/style.css:992]: - [ ] Great job on implementing the popup window functionality I liked it 💯👌 there is a small thing in your design based on your [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) the text inside the button should be White, so kindly add a `color: #fff` property to `btn-green` class 😄

Figma Design:
image

Your Design:
image


>

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @username-forGithub and @ehma90 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No linter errors ✅
- Your project is professional ✅
- Your repo is professional ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi Team 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**